releasenotes-builder/src/test/resources/com/github/checkstyle/xdocBug.txt

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
You removing space here

Reply:
@rnveach please help, maybe you have idea why it is happening.

Reply:
This was introduced in ec2026c1dcaa9f3d1eb3b6b78b675d46e6cd3966

Reply:
Got that!